master/internal/stream/system.go

```diff
@@ -629,12 +646,12 @@ func (ss *SubscriptionSet) SubscriptionMod(ctx context.Context, msg Subscription
 	var msgs []interface{}
 	var err error
 	msgs, err = subMod(ctx, msgs, err, ss.Trials, add.Trials, drop.Trials)
-	// msgs, err = subMod(msgs, err, ctx, ss.Experiments, add.Experiments, drop.Experiments)
+	msgs, err = subMod(ctx, msgs, err, ss.Experiments, add.Experiments, drop.Experiments)
```

Comment:
nit (none-blocking): while we're still adding more types, it might be worthwhile to keep some version of this comment in place to tag areas in the code that need to be updated